Assets/Shaders/SplatCube.shader

```diff
@@ -0,0 +1,207 @@
+Shader "Custom/CubeShader"
+{
+    Properties
+    {
+        // splat map
+        _SliceCount ("SliceCount", Range(1,256)) = 256
+        _SplatMap ("SplatMap", 2D) = "black" {}
+
+        // detail maps
+        _DetailUvScale ("DetailUvScale", Float) = 5.0
+        _AlbedoMaps ("AlbedoMaps", 2DArray) = "" {}
+        _NormalMaps ("NormalMaps", 2DArray) = "" {}
+        _RoughMaps ("RoughMaps", 2DArray) = "" {}
+        _CavityMaps ("CavityMaps", 2DArray) = "" {}
+
+        // noise map
+        _UvNoiseScale("UvNoiseScale", Float) = 15.0
+        _UvNoiseRadius("UvNoiseRadius", Float) = 5.0
+        _NoiseMap("NoiseMap", 2D) = "black" {}
+    }
+
+    SubShader
+    {
+        Tags { "RenderType"="Opaque" }
+
+        // shader program starts here
+        CGPROGRAM
+
+        #pragma only_renderers d3d11
+        // use PBR standard lighting model + enable shadows on all light types
+        #pragma surface surf Standard fullforwardshadows vertex:vert
+        //  only compile shader on platforms where texture arrays are available
+        #pragma require 2darray
+        // use shader model 3.0 target, to get nicer looking lighting
+        #pragma target 3.0
+
+        /*********************************
+        *          Vertex shader         *
+        *********************************/
+        float _DetailUvScale;
+
+        struct Input
+        {
+            float2 baseUv;
+            float2 detailUv;
+        };
+
+        void vert(inout appdata_full _vertex, out Input varyings_)
+        {
+            varyings_.baseUv = _vertex.texcoord.xy;
+            varyings_.detailUv = _vertex.texcoord.xy * _DetailUvScale;
+        }
+
+        /*********************************
+        *         Fragment shader        *
+        *********************************/
+        float _UvNoiseScale;
+        float _UvNoiseRadius;
+        sampler2D _NoiseMap;
```

Comment:
Will be removed

Reply:
https://github.com/davidasberg/cubes_traveller/pull/11/commits/08b629e7dd2ecaeffb23cb3b893479de655ce7ed